src/debug_app.py

```
@@ -0,0 +1,86 @@
+import debugpy # noqa
+import os
+import logging
+import threading
+import time
+
+logging.info("🔍 Starting debug_app module")
+
+if os.getenv("DEBUGPY_LOG_DIR"):
+    try:
+        logging.info(f"🪛 local debugpy log dir: {os.getenv('DEBUGPY_LOG_DIR')}")
+    except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)